supervisor/homeassistant/secrets.py

```diff
@@ -61,4 +61,4 @@ async def _read_secrets(self):
         except (YAMLError, AttributeError) as err:
             _LOGGER.error("Can't process Home Assistant secrets: %s", err)
         else:
-            _LOGGER.debug("Reloading Home Assistant secrets: %s", len(self.secrets))
+            _LOGGER.info("Reloading Home Assistant secrets: %s", len(self.secrets))
```

Comment:
Hmm, do we really want to show this every single time by default?

Reply:
This will help users not running debug see that it actually is happening.

Reply:
Yes, I get that, but my worry is, is that this will spam the logs.

Reply:
Well, it was over 1.5 year on info before...

Reply:
True, so the throttle isn't needed anymore?